clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -114,7 +118,8 @@ export async function activate(
     await initialize(context);
   initSmartTab(context);
 
-  TelemetryService.registerEvent(
+  await initTelemetry(context);
+  registerEvent(
```

Comment:
I think this should be moved to the top of the function, so that e.g. the exception reporting is available as soon as possible.